libraries/rush-lib/src/cli/actions/InitSubspaceAction.ts

```diff
@@ -70,9 +74,14 @@ export class InitSubspaceAction extends BaseRushAction {
     await Async.forEachAsync(
       templateFilePaths,
       async (templateFilePath) => {
-        const sourcePath: string = `${assetsSubfolder}/common/config/rush/${templateFilePath}`;
+        const defaultAssetSourcePath: string = `${defaultAssetsSubfolder}/common/config/rush/${templateFilePath}`;
         const destinationPath: string = `${subspaceConfigPath}/${templateFilePath.replace('[dot]', '.')}`;
-        await copyTemplateFileAsync(sourcePath, destinationPath, true);
+        await copyTemplateFileAsync(defaultAssetSourcePath, destinationPath, true);
+        if (userDefinedAssetsFolder) {
+          // if user provided their own assets file for subspace initiation
+          // we just copy and overwrite the default files
+          await copyTemplateFileAsync(userDefinedAssetsFolder, destinationPath, true);
```

Comment:
This approach seems like it will (1) try to copy a file that doesn't exist and (2) print a notice like `Overwriting: .pnpmfile.js` when really it is just overwriting its own file that it wrote on line 79.

Both issues could be avoided using `FileSystem.existsAsync()`.

Reply:
Also, `userDefinedAssetsFolder` is the path of a directory. Shouldn't `copyTemplateFileAsync()` receive a file path?

Please fill out the **How it was tested** PR notes 😸